Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+# Event Handling Workflows
+
+## Workflows Overview
+These flows are optional unless flow admin needs to build custom logic like displaying a Screen pop on the Agent's desktop etc. on occurrence of certain events. With v3.0 flows there will be default handling for all the events like
+adding participant , removing participant , closing the conversation etc. If flow admin needs any alternative handling or additional business logic then these flows can be imported and customized as per the requirement.
+These event handling flows are media channel agnostic and if different handling is required for channels or assets, then one can deploy as many flows as per the use case.
+
+### Routed Event flow
+* This workflow will be triggered when an agent is successfully assigned to a contact. The event for this workflow is 'Task Routed'. So once the contact has been routed to the agent and the
+agent has clicked on the accept button then this workflow will be triggered.
+
+#### When do you need Routed Event Flow ?
+
+* Routed event flow is an optional flow .It is needed only when you want to do some customization like screen pop or http calls once the agent has clicked on the accept button.
+
+#### Start Node of Routed Event Flow
+
+<img width="800" alt="WxCC Task V2" src="../../images/TaskRouted.png">
+
+Under the configuration section , we can see the event selected is Task Routed so this flow will be triggered on the Task Routed event. We can use the available variables from the
+output variables section.
+
+If you want to run the Routed Event flow to run only for specific asset or channel then we need to enable the conditions checkbox and add condition to the "WxCC Task V2" Node.
+For adding conditions please refer to the [Adding Conditions In Flow](#adding-conditions-in-flow) section.
+
+There is no restriction on the number of Routed Event Flow. You can add as many Routed Event Flow you need if different handling is required for each channel or asset.
+But in this case you need to specify condition for each flow.
+
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+### Modified Event flow
+
+* This workflow will be triggered when an agent has done a conversation transfer request or a conference request. The event for this workflow is 'Task Modified'. So once the contact has been transferred to another agent or
```

Comment:
done -> performed 

This workflow will be triggered when an agent has performed a conversation transfer or a conference.